packages/components/tests/unit/chat/MarkdownCodeSnippet.spec.js

```diff
@@ -88,6 +88,6 @@ describe('components/MarkdownCodeSnippet.vue', () => {
       provide: { projectDirectories: ['/home/user/my-project'] },
     });
 
-    expect(wrapper.find('[data-cy="context-title"]').text()).toBe('app/models/user.rb');
+    expect(wrapper.find('[data-cy="context-title"]').text()).toContain('app/models/user.rb');
```

Comment:
Won't these toContain checks pass even if the text is not rendering correctly (i.e. has the leading punctation show up on the right)?

Reply:
Yes, though they passed before as well. I've added examples of the behavior to Storybook, so Chromatic will pick up any changes in the future should there be a regression.

Reply:
To be clear, the issue wasn't present in the text of the element, just in how it was rendered.